dev/SapphireTest.php

```diff
@@ -43,11 +43,6 @@ class SapphireTest extends PHPUnit_Framework_TestCase {
 	 */
 	protected $usesDatabase = null;
 
-	/**
-	 * @deprecated since version 4.0
-	 */
-	protected $originalMailer;
```

Comment:
It's ok if your fixed code doesn't rely on it, as long as third party tests (subclasses of SapphireTest) which once relied on these don't break.